src/test/java/com/google/ehub/servlets/ItemPageServletTest.java

```diff
@@ -0,0 +1,129 @@
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+@RunWith(JUnit4.class)
+public class ItemPageServletTest {
+  private final ItemPageServlet servlet = new ItemPageServlet();
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+  private static final String JSON_CONTENT_TYPE = "application/json";
+
+  private static final String TITLE = "Star Wars";
+  private static final String DESCRIPTION = "Blah....";
+  private static final String IMAGE_URL = "Image.png";
+  private static final String RELEASE_DATE = "09/26/1972";
+  private static final String RUNTIME = "2 hours";
+  private static final String GENRE = "Sci-Fi";
+  private static final String DIRECTORS = "George Lucas";
+  private static final String WRITERS = "George Lucas";
+  private static final String ACTORS = "Mark Hamill, Harrison Ford";
+  private static final String OMDB_ID = "tt23113212";
+
+  @Mock HttpServletRequest request;
+  @Mock HttpServletResponse response;
+  @Mock PrintWriter printWriter;
+
+  @Before
+  public void init() {
+    MockitoAnnotations.initMocks(this);
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  @Test
+  public void doGETReturnsItemPageData() throws IOException {
+    EntertainmentItem selectedItem = new EntertainmentItem.Builder()
+                                         .setTitle(TITLE)
+                                         .setDescription(DESCRIPTION)
+                                         .setImageUrl(IMAGE_URL)
+                                         .setReleaseDate(RELEASE_DATE)
+                                         .setRuntime(RUNTIME)
+                                         .setGenre(GENRE)
+                                         .setDirectors(DIRECTORS)
+                                         .setWriters(WRITERS)
+                                         .setActors(ACTORS)
+                                         .setOmdbId(OMDB_ID)
+                                         .build();
+    Key itemId = EntertainmentItemDatastore.getInstance().addItemToDatastore(selectedItem);
+    when(request.getParameter("itemId")).thenReturn(itemId.getId() + "");
+    when(response.getWriter()).thenReturn(printWriter);
+    CommentDataManager commentDataManager = new CommentDataManager();
+    commentDataManager.addItemComment(itemId.getId(), "Nice", 1093923L);
+
+    servlet.doGet(request, response);
+    CommentData comment = new CommentData(itemId.getId(), "Nice", 1093923L);
+    List<CommentData> comments = new ArrayList<>();
+    comments.add(comment);
+    Optional<EntertainmentItem> expectedItem =
+        EntertainmentItemDatastore.getInstance().queryItem(itemId.getId());
+    ItemPageData itemData = new ItemPageData(expectedItem.get(), comments);
```

Comment:
Nit: this can be cleaned up a little bit by using Lists.newArrayList ([from Guava library](https://guava.dev/releases/19.0/api/docs/com/google/common/collect/Lists.html))
```suggestion
    Optional<EntertainmentItem> expectedItem =
        EntertainmentItemDatastore.getInstance().queryItem(itemId.getId());
    ItemPageData itemData = new ItemPageData(expectedItem.get(), Lists.newArrayList(comment)));
```